source/getting-started/tutorials/sensors/index.md

```diff
@@ -0,0 +1,283 @@
+ - Sensor collection time for the ADC can be gathered with the hardware command `hw tm time adc` or the C function `timing_manager_get_time_per_sensor(ADC)`. It is around 0.86 microseconds.
+ - The control task time can be gathered with the user-made command `ctrl stats print`. We are specifically looking at the Run-Time.
+
+```
+Task Stats:
+Loop Num:	21431 samples
+Loop Min:	94.52 usec
+Loop Max:	106.21 usec
+Loop Mean:	100.00 usec
+Loop Var:	0.02 usec
+Run Num:	21513 samples
+Run Min:	3.25 usec
+Run Max:	3.92 usec
+Run Mean:	3.40 usec
+Run Var:	0.00 usec
+```
+
+The loop mean is is how much time there is between successive executions of the control task. It should be `1` / `TASK_CONTROLLER_UPDATES_PER_SEC`. And in this case it is, at `1` / `10000` seconds.
+
+### We can now draw our timing diagram with exact parameters:
+
+![](images/tmVSI.svg)
+
+We can see that we are sampling the sensors once per control task. That's because our User Event Ratio of `10` fits perfectly with the ratio between the PWM frequency and our control task's frequency. This is the gold standard. In this tutorial we will experiment with changes to the parameters of the timing manager and observe the effects on control task timings.
+s
+## Experiment 1 - Ratio is too large
+
+If we increase the User Event Ratio, we can cause the control task to run at less than 10Khz. Let's increase it to 20 by putting `timing_manager_set_ratio(20)` in the `controller_init()` function.
+
+`app_controller.c`:
+```
+void app_controller_init(void)
+{
+    // Enable data sampling for ADC
+    timing_manager_enable_sensor(ADC);
+    // set User Event Ratio
+    timing_manager_set_ratio(20);
+    // Register "ctrl" command with system
+    cmd_controller_register();
+}
+```
+What does this do? We've made it so that the sensors will collect data every 20 PWM cycles, and then the scheduler will run the control task. Since our PWM frequency is still 10KHz, that means our sensors will collect data every 200us. It also means that our control task can only run once every 200us, instead of 100us like it should.
+
+![](images/tmVSI20.svg)
+
+Rebuild and run the new program, and use the command `ctrl stats print` to view the loop time (after doing `ctrl init`).
+
+```
+Task Stats:
+Loop Num:	26806 samples
+Loop Min:	193.78 usec
+Loop Max:	206.22 usec
+Loop Mean:	200.00 usec
+Loop Var:	0.02 usec
+Run Num:	26932 samples
+Run Min:	3.25 usec
+Run Max:	4.23 usec
+Run Mean:	3.41 usec
+Run Var:	0.00 usec
+```
+
+The loop mean is how much time there is between successive executions of the control task. It should be `1` / `TASK_CONTROLLER_UPDATES_PER_SEC`, but in this case it is double of that. That indicates that the control task is only running at half of `TASK_CONTROLLER_UPDATES_PER_SEC`.
+
+Making the User Event Ratio too high is one way that control tasks can be slowed down past their target `TASK_CONTROLLER_UPDATES_PER_SEC`.
```

Comment:
Hmm... this might be a larger issue to review with @elsevers and maybe @npetersen2 . Is this intended behavior? I feel like the intended behavior should be that the control task still runs at 10kHz (every 100us) but the sensor data is updated every other control cycle, half as often, etc at 5kHz (every 200us).